### PR TITLE
Updated the documentation with the table step width option

### LINKED ISSUE #34: Add start:step:end operator to documentation
### DESCRIPTION
**Is your change request related to a problem? Please describe.**
The syntax `myTable(start:step:end)` can currently not be found in documentation.
Add the option (with an example) to the documentation for table and cluster (and possibly others if affected).
The keyword `range` could also be considered for additional reference.

**Describe the solution you'd like**
Adapt documentation.

**Describe possible alternatives you've considered**
Add a documentation article.

**Additional context**
Relates to the issue #33, where such a feature that already exists was requested.

(*Do not write below this line*)

-----------------------------------------------------
### ANALYSIS
This operator is already part of the documentation (See `help operator`), so find out, where a reference is missing and add it there.

### IMPLEMENTATION STEPS
(*see also our [Wiki for implementation guidelines](https://github.com/numeredev/NumeRe/wiki/HowTo:-Contribute-Code)*)
- Implement the necessary changes in a new branch created here on GitHub
- Test your implementation

### DOCUMENTATION STEPS
(*see also our [Wiki for further information](https://github.com/numeredev/NumeRe/wiki/HowTo:-Create-Language-Strings)*)
- Update the changes log
- Add comments to your implementation
- Add Doxygen documentation comments
- Create or update the documentation articles (`*.NHLP` and `*.NDB` files, if needed)
- Update the language strings (`*.NLNG` files, if needed)

### PULL REQUEST
- Create a pull request for your changes
- Fill out the template
- Assign @numere-org/maintainers as reviewers

### PR DESCRIPTION
**Pull request by raphael-sys**
Possibility to use a range with a specified step width as requested in ticket numere-org/NumeRe#34

Please review if keywords need to be updated in other documents too or if this is enough already.

### SUMMARY
Implements necessary language changes.
Resolves numere-org/NumeRe#34

*Reviewers:* @numere-org/maintainers

### DOCUMENTATION
* **Documentation articles:**
    * [x] corresponding documentation articles updated
    * [ ] new documentation articles created
    * [ ] not needed
* **Language files:**
    * [ ] corresponding language files updated
    * [x] not needed

### CODE REVIEW

File: de-DE/docs/cache.nhlp [hunk 1]
```diff
@@ -32,10 +32,12 @@
 		<keyword>cols</keyword>
 		<keyword>ncols</keyword>
 		<keyword>every</keyword>
+		<keyword>range</keyword>
 	</keywords>
 	<contents>
 		<syntax>
 			{VAL} = TABLE(i1:i2,j1:j2)
+			{VAL} = TABLE(START:STEP:END,:)
```
Comment (numeredev) [inline de-DE/docs/cache.nhlp]:
```suggestion
			{VAL} = TABLE({START:STEP:END},:)

```
Needs the additional brace

File: de-DE/docs/cache.nhlp [hunk 2]
```diff
@@ -65,6 +67,7 @@
 		Standardm��ig existiert die Tabelle <code>table()</code>. Mit dem Kommando <code>new</code> k�nnen aber weitere Tabellen mit anderen Namen erzeugt werden:<codeblock>new TABLE1(), TABLE2(), ...</codeblock>
 		Einzelne Elemente einer Tabelle k�nnen mit der Syntax <code>TABLE(ZEILE, SPALTE)</code> ausgelesen werden. Ebenso k�nnen damit auch Werte in die Tabelle geschrieben werden. Dabei sind Tabellen wie gew�hnliche Variablen zu behandeln. Die Indizierung erfolgt dabei durch nat�rliche Zahlen <em>gr��er</em> Null: 1,2,3,...
 		Bereiche von Elementen k�nnen dabei mit der Syntax <code>ERSTER INDEX:ZWEITER INDEX</code> erhalten werden. Wird dabei einer der beiden Indices weggelassen, wird er automatisch durch den minimalen bzw. maximalen Index erg�nzt. Fehlen beide Indices, wird folglich der gesamte, m�gliche Bereich ausgew�hlt. Es k�nnen auch Zeilen/Spalten aus anderen Tabellen oder Datenfiles als Indexmenge verwendet werden.
+		Zus�tzlich kann mittels <code>ERSTER INDEX:SCHRITTWEITE:LETZTER INDEX</code> die Schrittweite der Indizes vorgegeben werden.
```
Comment (numeredev) [inline de-DE/docs/cache.nhlp]:
```suggestion
		Zus�tzlich kann mittels <a href="nhlp://operator?frame=new"><code>{ERSTER INDEX:SCHRITTWEITE:LETZTER INDEX}</code></a> die Schrittweite der Indizes vorgegeben werden.

```

File: en-GB/docs/cache.nhlp [hunk 3]
```diff
@@ -32,10 +32,12 @@
 		<keyword>cols</keyword>
 		<keyword>ncols</keyword>
 		<keyword>every</keyword>
+		<keyword>range</keyword>
 	</keywords>
 	<contents>
 		<syntax>
 			{VAL} = TABLE(i1:i2,j1:j2)
+			{VAL} = TABLE(START:STEP:END,:)
```
Comment (numeredev) [inline en-GB/docs/cache.nhlp]:
```suggestion
			{VAL} = TABLE({START:STEP:END},:)

```

File: en-GB/docs/cache.nhlp [hunk 4]
```diff
@@ -65,6 +67,7 @@
 		By default, the table <code>table()</code> exists. With the command <code>new</code>, however, further tables with other names can be created:<codeblock>new TABLE1(), TABLE2(), ...</codeblock>.
 		Individual elements of a table can be read out with the syntax <code>TABLE(ROW, COLUMN)</code>. Likewise, values can also be written into the table with it. Tables are to be treated like ordinary variables. The indexing is done by natural numbers <em>greater than</em> zero: 1,2,3,...
 		Ranges of elements can be obtained with the syntax <code>FIRST INDEX:SECOND INDEX</code>. If one of the two indices is omitted, it is automatically supplemented by the minimum or maximum index. If both indices are missing, the entire possible range is selected. Rows/columns from other tables or data files can also be used as index sets.
+		Additionally, the step size of the indices can be specified using <code>FIRST INDEX:STEP WIDTH:LAST INDEX</code>.
```
Comment (numeredev) [inline en-GB/docs/cache.nhlp]:
```suggestion
		Additionally, the step size of the indices can be specified using <a href="nhlp://operator?frame=new"><code>{FIRST INDEX:STEP WIDTH:LAST INDEX}</code></a>.

```

File: en-GB/docs/find.ndb [hunk 5]
```diff
@@ -427,7 +427,7 @@ TAB().scorrof(n,m,{r1},{r2})~Calculates the Spearman correlation of the columns~
 TAB().size~Determines the size of the table~dimension size element count bounding rectangle~.size
 TAB().typeof({n})~Returns the types of the columns as strings~typeof() convert() konvertieren category string datetime logical value data type~.typeof(
 TAB().zscoreof(n,{r})~Calculates the standardization of the values related to their deviation from the mean~zscoreof() std() avg() mean average standarddeviation zscale~.zscoreof(
-TABLE(i,j)~Table object with read/write access to the element "i,j"~zwischenspeicher save cache() table() sortieren swap rename~help table()
+TABLE(i,j)~Table object with read/write access to the element "i,j"~zwischenspeicher save cache() table() sortieren swap rename range step widht~help table()
```
Comment (numeredev) [inline en-GB/docs/find.ndb]:
```suggestion
TABLE(i,j)~Table object with read/write access to the element "i,j"~zwischenspeicher save cache() table() sortieren swap rename range step width~help table()

```

**Review by numeredev — CHANGES_REQUESTED** [review]
Please change the mentioned items. You might as well add similar information to the `cluster.nhlp` files as those also benefit of `CLST{{i:s:e}}` syntax (which, btw., also supports things like `{a, b, c, D:E, f, G:H:I}`)

**Comment by raphael-sys** [comment]
@numeredev: added the requested `clst` references as syntax examples in the help article and a keyword in the find database.

**Comment by raphael-sys** [comment]
It seems like the format of the file again is broken. I will look into why this keeps happening.

**Comment by raphael-sys** [comment]
Also added a missing find entry for the cluster article

**Review by numeredev — APPROVED** [review]
Changes approved. Can be merged.

**Pull request merged**